config/owasp/suppressions.xml

```diff
@@ -1,36 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
 
+  <!--Please add all the false positives under the below section-->
   <suppress>
-    <notes><![CDATA[
-     Suppressing as it's a false positive (see: https://pivotal.io/security/cve-2018-1258)
-   ]]></notes>
-    <gav regex="true">^org\.springframework\.(security|boot):(spring-security-crypto:5.6.[0-5]|spring-boot-starter-oauth2-(client|resource-server):2.6.[0-8])</gav>
-    <cpe>cpe:/a:pivotal_software:spring_security</cpe>
-    <cve>CVE-2018-1258</cve>
-  </suppress>
-
-  <suppress>
-    <notes>
-      False-positive. See CCD-3255 (CCD-3251 - CCD-3258).
+    <notes>False Positives
+      CVE-2016-1000027 refer https://tools.hmcts.net/jira/browse/WLTS-35
     </notes>
     <cve>CVE-2016-1000027</cve>
   </suppress>
+  <!--End of false positives section -->
 
   <suppress>
     <notes><![CDATA[
-     Temp suppression; see OSSIndex/vulns#276 & jeremylong/DependencyCheck#4528.
-   ]]></notes>
-    <gav regex="true">^org\.springframework\.security:spring-security-crypto:5\.6\.[0-5]</gav>
-    <cve>CVE-2020-5408</cve>
+      Suppressing as it's a false positive (see: https://pivotal.io/security/cve-2018-1258)
+      ]]>
+      https://tools.hmcts.net/jira/browse/WLTS-36
+    </notes>
+    <gav regex="true">^org\.springframework\.(security|boot):(spring-security-crypto:5.6.[0-5]|spring-boot-starter-oauth2-(client|resource-server):2.6.[0-8])</gav>
+    <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+    <cve>CVE-2018-1258</cve>
```

Comment:
move this to false positive section

Reply:
i don't know how i can it has this element <cpe>cpe:/a:pivotal_software:spring_security</cpe>